integrations/arize.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: integration
+name: Arize AI
+description: Monitor and trace your Haystack pipelines Arize and Phoenix
+authors:
+    - name: Arize AI
+      socials:
+        github: Arize-ai
+        twitter: arizeai
+        linkedin: arizeai
+pypi: https://pypi.org/project/openinference-instrumentation-haystack/
+repo: https://github.com/Arize-ai/openinference
+type: Monitoring Tool
+report_issue: https://github.com/Arize-ai/openinference/issues
+logo: /logos/arize.png
+version: Haystack 2.0
+toc: true
+---
+### **Table of Contents**
+- [Overview](#overview)
+- [Features](#features)
+- [Installation](#installation)
+- [Usage](#usage)
+- [License](#license)
+
+## Overview
+
+Arize is AI Observability and Evaluation platform designed to help you troubleshoot, evaluate, and experiment on LLM and ML applications. Developers use Arize to get applications working quickly, evaluate performance, detect and prevent production issues, and curate datasets.
+
+Phoenix is Arize's open-source library. Phoenix focuses specifically on LLM applications, and provides similar troubleshooting, evaluate, and experimentation functionality as Arize.
+
+You can bring your Haystack pipelines to both Arize and Arize Phoenix (the OSS offering), with options on either using the Phoenix API or opting for the self-hosted version.
+
+- [Documentation for Arize AI + Haystack](https://docs.arize.com/arize/large-language-models/tracing/auto-instrumentation/haystack)
+- [Documentation for Phoenix + Haystack](https://docs.arize.com/phoenix/tracing/integrations-tracing/haystack)
+
+## Installation
+
+```bash
+pip install openinference-instrumentation-haystack haystack-ai arize-otel opentelemetry-sdk opentelemetry-exporter-otlp
+```
+
+## Usage
+### Setting up the `HaystackInstrumentor`
+
+To trace any Haystack pipeline with Phoenix, you have to create a `HaystackInstrumentor`. Haystack pipelines that run within the same environment send traces to Phoenix.
```

Comment:
To trace any Haystack pipeline with Phoenix, you have to create a `HaystackInstrumentor`. Haystack pipelines that run within the same environment send traces to Phoenix.
The easiest option is to [launch a hosted phoenix instance](https://phoenix.arize.com/). Once your instance is provisioned, you'll be able to grab the API Key 

```python
from arize_otel import register_otel, Endpoints

# Setup OTEL tracing for hosted Phoenix
register_otel(
    endpoints=[Endpoints.HOSTED_PHOENIX],
    api_key="YOUR_HOSTED_PHOENIX_API_KEY"
)

HaystackInstrumentor().instrument()
```
There is also an option to self-host phoenix if you prefer [here](https://docs.arize.com/phoenix/deployment/deploying-phoenix)

Reply:
I think the above suggestion would be easier to follow for users!  The existing code block you had was specifically for Arize platform so users would've gotten stuck if they just copy pasted.  this version is a lot more copy paste friendly